catalog/ui/catalog-ui-search/src/main/webapp/component/filter/filter.view.js

```diff
@@ -78,6 +78,7 @@ define([
                 }).map(function(metacardType){
                     return {
                         label: metacardType.alias || metacardType.id,
+                        description: (properties.attributeDescriptions || {})[metacardType.id],
```

Comment:
This is used in a `div.title`. Correct me if I'm wrong, but I think `{}[metacardType.id]` will become "`undefined`" in the `div.title`. Is this the desired behavior? Maybe use the following instead?
`description: properties.attributeDescriptions[metacardType.id] || "",`

Reply:
This is intentional/desired. As written it handles the case where `properties.attributeDescriptions` is `undefined`, which would throw an error in your snippet. Also handlebars resolves the undefined to an empty string when rendering.

Reply:
Fair point. I should have done the following instead. Anyway, if undefined is rendered as empty string, it should be fine.
`description: (properties.attributeDescriptions || {})[metacardType.id] || "",`